src/frontend/fluent-core/Frontend.ts

```diff
@@ -0,0 +1,17 @@
+import { provideFluentDesignSystem, allComponents as allFluentComponents } from '@fluentui/web-components';
+provideFluentDesignSystem().withShadowRootMode('closed').register(allFluentComponents);
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */ //=> Import necessary to register web-components
+import * as _ from './components/_index';
+import type { IWindowController } from '../WindowController';
+import type { IFrontendModule } from '../IFrontend';
+import App from './App';
+
+class BarelyFluent implements IFrontendModule {
+    async Render(root: HTMLElement, windowController: IWindowController): Promise<void> {
+        const _foo = windowController;
```

Comment:
Use [FAST DI Container](https://www.fast.design/docs/apps-and-experiences/dependency-injection) or move to [StateService.ts](https://github.com/manga-download/haruneko/pull/53/files#diff-488cba7ae975394ca97a92f0e8e5105454897a8fcac476d378bc51302ae6ac17)

Reply:
Solved in 672f6f54